calendar_reboot.js

```diff
@@ -265,27 +260,34 @@ $(document).ready(function() {
     
     
     //WHEN PAGE LOADS
-    //first load the calendarUniqueId from storage.
-    calendarUniqueId = loadFromLocalStorage(masterKey);
+    //first load the allCalendarIds from storage.
+    allCalendarIds = store.getAllCalendarIds();
     //if it's not in localstorage make it an empty object
-    if (calendarUniqueId === null) {
-        calendarUniqueId = {};
+    if (allCalendarIds === null) {
+        allCalendarIds = {};
     }
     
-    //GOING THROUGH THE KEYS OF THE DICTIONARY calendarUniqueId
-    for (var key in calendarUniqueId) {
-      if (calendarUniqueId.hasOwnProperty(key)) {
+    //GOING THROUGH THE KEYS OF THE DICTIONARY allCalendarIds
+    for (var key in allCalendarIds) {
+      if (allCalendarIds.hasOwnProperty(key)) {
           if (key !== current_active_calendar) {
-            addCalendarToDropdown(key, calendarUniqueId[key]);
+            addCalendarToDropdown(key, allCalendarIds[key]);
           }
       }
     }
     
+    //get the current_active calendar id from storage if any
+    var activeCalendarId = store.getActive();
     
-    var activeCalendar = loadFromLocalStorage(current_active_calendar);
-    
-    if (activeCalendar !== null) {
-        displayCalendar(activeCalendar);
+    if (activeCalendarId !== null) {
+        console.log("the activeCalendarKey was not null");
```

Comment:
Probably remove this. Maybe replace with a more useful message instead. `"Loading active calendar <...>"`